data/json/items/vehicle/manual.json

```
@@ -12,16 +12,13 @@
     "volume": "500 ml",
     "category": "veh_parts",
     "price": "90 USD",
-    "price_postapoc": "1 USD",
-    "melee_damage": { "bash": 10 }
+    "price_postapoc": "1 USD"
```

```suggestion
    "price_postapoc": "1 USD",
    "melee_damage": { "bash": 10 }
```

This has to come back or the tests will not let this go through. Why is this changed here anyway?

---

because their dps test was already borked, i assumed hand rims is a bad weapon to be actually listed in dps test

---

> because their dps test was already borked, i assumed hand rims is a bad weapon to be actually listed in dps test

I don't disagree that they shouldn't be there, but the damage alone should probably not be scope creeped into this PR either...